Wrapping/Generators/Python/itk/support/extras.py

```diff
@@ -1165,6 +1165,24 @@ def imwrite(
     The writer is instantiated with the image type of the image in
     parameter (or, again, with the output image of the filter in parameter).
     """
+
+    # Check if output path exists
+    msg = ""
+    if not os.path.isdir(os.path.dirname(filename)):
+        msg += "\nThe output dir doesn't exist. \n" + f"Filename = {filename}"
+        raise RuntimeError(
+            f"Could not create IO object for writing file {filename}" + msg
+        )
```

Comment:
This would be a useful check to occur in the C++ code.

Reply:
I agree, but I assume that performing an equivalent check in C++ would be a lot harder. If the author has the will to explore this, it would be good.